src/main/resources/config/liquibase/changelog/20210125101100_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="b-fein" id="20210125101100">
+        <addColumn tableName="programming_exercise_test_case">
+            <column name="visibility" type="VARCHAR(255)"/>
+        </addColumn>
+        <sql>
+            update programming_exercise_test_case set visibility = 'ALWAYS' where after_due_date = false or after_due_date is null;
+            update programming_exercise_test_case set visibility = 'AFTER_DUE_DATE' where after_due_date = true
+        </sql>
+        <dropColumn tableName="programming_exercise_test_case" columnName="after_due_date"/>
```

Comment:
`dropColumn` is a breaking change on the test server and leads to issues if a version before this change is deployed. Can you please remove it for now (and not deploy to test servers any more before removal?)
We should add this in a small follow-up PR